blocks/library/cover-image/index.js

```diff
@@ -52,7 +52,7 @@ registerBlockType( 'core/cover-image', {
 					<BlockAlignmentToolbar
 						value={ align }
 						onChange={ updateAlignment }
-						controls={ validAlignments }
+						controls={ [ 'left', 'center', 'right' ].concat( settings.wideImages ? [ 'wide', 'full' ] : [] ) }
```

Comment:
Should we bake this into the `BlockAlignmentToolbar` component, since it already has some knowledge of wide and full widths? Maybe as a boolean prop.